src/pages/DirectMessage.tsx

```diff
@@ -0,0 +1,252 @@
+import {
+  Avatar,
+  Box,
+  Divider,
+  InputAdornment,
+  List,
+  ListItem,
+  ListItemText,
+  Stack,
+  TextField,
+  Typography,
+} from "@mui/material";
+import IconButton from "@mui/material/IconButton";
+import { useEffect, useRef, useState } from "react";
+import axios from "axios";
+import { useParams } from "react-router-dom";
+import { useAppDispatch, useAppSelector } from "../state/hooks";
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
+import AddPhotoAlternateOutlinedIcon from "@mui/icons-material/AddPhotoAlternateOutlined";
+import EmojiEmotionsOutlinedIcon from "@mui/icons-material/EmojiEmotionsOutlined";
+import GifBoxOutlinedIcon from "@mui/icons-material/GifBoxOutlined";
+import SendIcon from "@mui/icons-material/Send";
+import ConversationList from "../components/Messages/ConversationList";
+import {
+  setSelectedConversation,
+  updateConversation,
+} from "../state/slices/messagesSlice";
+
+const styles = {
+  directMessageActivityContainer: {
+    width: "50%",
+    display: "flex",
+    flexDirection: "column",
+    height: "100%",
+  },
+  directMessageActivityHeader: {
+    display: "flex",
+    justifyContent: "space-between",
+    paddingLeft: 2,
+    paddingRight: 2,
+  },
+  message: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "flex-start",
+  },
+  messageText: {
+    padding: 1,
+    borderRadius: 10,
+    backgroundColor: "#cce3d9",
```

Comment:
Adding a colour in `Theme.tsx` for this colour and setting it to that would be ideal. We can talk about this